docs/results/node-20.json

```diff
@@ -1 +1 @@
mark":"assertLoose","name":"tson","ops":732784,"margin":0.29,"nodeVersion":"v20.11.0"},{"benchmark":"assertStrict","name":"tson","ops":723310,"margin":0.36,"nodeVersion":"v20.11.0"},{"benchmark":"assertLoose","name":"@typeofweb/schema","ops":1651915,"margin":0.65,"nodeVersion":"v20.11.0"},{"benchmark":"parseStrict","name":"@typeofweb/schema","ops":1634465,"margin":1.5,"nodeVersion":"v20.11.0"},{"benchmark":"parseSafe","name":"typia","ops":61492853,"margin":0.32,"nodeVersion":"v20.11.0"},{"benchmark":"parseStrict","name":"typia","ops":38320332,"margin":0.54,"nodeVersion":"v20.11.0"},{"benchmark":"assertStrict","name":"typia","ops":36180118,"margin":0.43,"nodeVersion":"v20.11.0"},{"benchmark":"assertLoose","name":"typia","ops":140528530,"margin":0.71,"nodeVersion":"v20.11.0"},{"benchmark":"parseStrict","name":"unknownutil","ops":3597629,"margin":0.68,"nodeVersion":"v20.11.0"},{"benchmark":"assertStrict","name":"unknownutil","ops":3520788,"margin":0.58,"nodeVersion":"v20.11.0"},{"benchmark":"assertLoose","name":"unknownutil","ops":4671982,"margin":0.23,"nodeVersion":"v20.11.0"},{"benchmark":"assertLoose","name":"valibot","ops":2316292,"margin":0.38,"nodeVersion":"v20.11.0"},{"benchmark":"assertStrict","name":"valibot","ops":1654580,"margin":0.55,"nodeVersion":"v20.11.0"},{"benchmark":"parseSafe","name":"valibot","ops":2261959,"margin":0.38,"nodeVersion":"v20.11.0"},{"benchmark":"parseStrict","name":"valibot","ops":1655215,"margin":1.01,"nodeVersion":"v20.11.0"},{"benchmark":"parseSafe","name":"valita","ops":6928062,"margin":0.2,"nodeVersion":"v20.11.0"},{"benchmark":"parseStrict","name":"valita","ops":6912033,"margin":0.31,"nodeVersion":"v20.11.0"},{"benchmark":"assertLoose","name":"valita","ops":6934665,"margin":1.66,"nodeVersion":"v20.11.0"},{"benchmark":"assertStrict","name":"valita","ops":6610081,"margin":0.47,"nodeVersion":"v20.11.0"},{"benchmark":"parseSafe","name":"vality","ops":215435,"margin":0.32,"nodeVersion":"v20.11.0"},{"benchmark":"parseStrict","name":"vality","ops":209876,"margin":0.2,"nodeVersion":"v20.11.0"},{"benchmark":"assertLoose","name":"vality","ops":213104,"margin":0.29,"nodeVersion":"v20.11.0"},{"benchmark":"assertStrict","name":"vality","ops":209316,"margin":0.34,"nodeVersion":"v20.11.0"},{"benchmark":"assertLoose","name":"yup","ops":48376,"margin":0.41,"nodeVersion":"v20.11.0"},{"benchmark":"parseSafe","name":"yup","ops":47032,"margin":0.25,"nodeVersion":"v20.11.0"},{"benchmark":"parseSafe","name":"zod","ops":755162,"margin":0.25,"nodeVersion":"v20.11.0"},{"benchmark":"parseStrict","name":"zod","ops":703746,"margin":0.36,"nodeVersion":"v20.11.0"},{"benchmark":"assertLoose","name":"zod","ops":656816,"margin":1.44,"nodeVersion":"v20.11.0"},{"benchmark":"assertStrict","name":"zod","ops":668024,"margin":0.28,"nodeVersion":"v20.11.0"},{"benchmark":"parseSafe","name":"deepkit","ops":6491176,"margin":0.28,"nodeVersion":"v20.11.0"},{"benchmark":"assertLoose","name":"deepkit","ops":39182400,"margin":0.24,"nodeVersion":"v20.11.0"}]}
```

Comment:
Please rm benchmark results from the PR. They'll be generated automatically after merge.